sdks/python/apache_beam/io/gcp/bigquery_read_it_test.py

```diff
@@ -162,6 +163,15 @@ def test_iobase_source(self):
               query=self.query, use_standard_sql=True, project=self.project))
       assert_that(result, equal_to(self.TABLE_DATA))
 
+  @attr('IT')
+  def test_valueprovider_query_string(self):
+    query = StaticValueProvider(str, self.query)
+    with beam.Pipeline(argv=self.args) as p:
+      result = (
+          p | 'read' >> beam.io._ReadFromBigQuery(
+              query=query, use_standard_sql=True, project=self.project))
+      assert_that(result, equal_to(self.TABLE_DATA))
```

Comment:
could you use the value provider in another one of the tests in this file? Since each test in this file takes ~5min to run, I'd prefer to avoid adding a new test.